config/install/core.entity_view_display.node.stanford_publication.default.yml

```diff
@@ -27,48 +27,42 @@ third_party_settings:
           columns: default
           label: ''
         components:
-          1a0c44d4-f6a2-457f-9d43-0bfb7f32c58f:
-            uuid: 1a0c44d4-f6a2-457f-9d43-0bfb7f32c58f
+          4df7625d-a934-4076-a6ff-d70bf5aa9f2e:
```

Comment:
Is this a situation where the UUID can, in fact, be safely changed?

Reply:
yeah. the `uuid` at the very top of the file and the base level of the yaml structure is the only important one.